src/lib/linux-api/src/signal.rs

```diff
@@ -1175,6 +1184,17 @@ mod export {
         unsafe { SigInfo::peel(SigInfo::new_for_kill(signal, sender_pid, sender_uid)) }
     }
 
+    #[no_mangle]
+    pub unsafe extern "C" fn linux_kill(pid: i32, sig: i32) -> i32 {
+        let Ok(sig) = Signal::try_from(sig) else {
+            return Errno::EINVAL.to_negated_i32()
+        };
```

Comment:
Not really relevant for this PR, but it seems like it would be more useful to have this the other way around. To have the rust function take a `pid: i32, sig: i32` and this function would pass those directly. And maybe have a rust wrapper that takes a `Signal`. For example it might be nice in the future to use this library in the shadow integration tests, but we can't because they don't allow a signal that isn't known by the library. For example there's no way to do the equivalent of the C code: `kill(0, INT_MAX)`. This rust wrapper would return `EINVAL` and it would never make it to the kernel, even if that's not what the kernel would return.